cmd/installer/pkg/imager/input.go

```diff
@@ -0,0 +1,39 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package imager
+
+// Input describes inputs for image generation.
+type Input struct {
+	// Kernel is a vmlinuz file.
+	Kernel FileAsset `yaml:"kernel"`
+	// Initramfs is a initramfs file (without system extensions).
+	Initramfs FileAsset `yaml:"initramfs"`
+	// SecureBoot is a section with secureboot keys, only for SecureBoot enabled builds.
+	SecureBoot *SecureBootAssets `yaml:"secureboot,omitempty"`
+	// SystemExtensions is a list of system extensions to install.
+	SystemExtensions []ContainerAsset `yaml:"systemExtensions,omitempty"`
+}
+
+// FileAsset describes a file asset.
+type FileAsset struct {
+	// Path to the file.
+	Path string `yaml:"path"`
+}
+
+// ContainerAsset describes a container asset.
+type ContainerAsset struct {
+	// ImageRef is a reference to the container image.
+	ImageRef string `yaml:"imageRef"`
+}
+
+// SecureBootAssets describes secureboot assets.
+type SecureBootAssets struct {
+	// SecureBoot signing key & cert.
+	SigningKeyPath  string `yaml:"signingKeyPath"`
+	SigningCertPath string `yaml:"signingCertPath"`
+	// PCR signing key & public key.
+	PCRSigningKeyPath string `yaml:"pcrSigningKeyPath"`
+	PCRPublicKeyPath  string `yaml:"pcrPublicKeyPath"`
```

Comment:
don;t we need the auto-enroll db file too?

Reply:
yeah, we need that, I'm still updating things, not sure where to put it yet (as it's ISO-specific).

Reply:
just a note, aws/cloud providers needs a different format though (we might need it eventually) https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/uefi-secure-boot-optionB.html